gk/main.c

```diff
@@ -909,6 +1086,10 @@ process_pkts_front(uint8_t port_front, uint8_t port_back,
 			switch (fib->action) {
 			case GK_FWD_GRANTOR:
 				/*
+				 * We double the hash table size to
+				 * alleviate memory pressure
+				 * when the table is full.
```

Comment:
I'm concerned this is not the functionality that we want. If there is pressure on memory because of a full table, wouldn't expanding it allow more entries to take up more memory? This would allow the hash table to grow indefinitely, no?